docs/howtoguides/configure_proxies.md

```diff
@@ -1,17 +1,100 @@
+required for enabling Ubuntu Advantage services. This includes APT, Snaps, and Livepatch.
 
-In addition, the UA Client will automatically set up proxies for all programs required for enabling Ubuntu Advantage services. This includes APT, Snaps, and Livepatch.
+## HTTP/HTTPS Proxies
 
-Proxies can be set using the `ua config set` command.
+To configure standard HTTP and/or HTTPS proxies, run the following commands:
 
-HTTP/HTTPS proxies are set using the fields `http_proxy` and `https_proxy`, respectively. The values for these fields will also be used for Snap and Livepatch proxies.
+```console
+$ sudo ua config set http\_proxy=http://host:port
+$ sudo ua config set https\_proxy=https://host:port
+```
 
-APT proxies are defined separately. You can set global apt proxies that affect the whole system using the fields `apt_http_proxy` and `apt_https_proxy`.
+After running the above commands, UA client:
 
-> Starting in to-be-released Version 27.9, APT proxies config options will change. You will be able to set global apt proxies that affect the whole system using the fields `global_apt_http_proxy` and `global_apt_https_proxy`. Alternatively, you could set apt proxies only for UA related services with the fields `ua_apt_http_proxy` and `ua_apt_https_proxy`.
+1. Verifies that the proxy is working by using it to reach `api.snapcraft.io`
+2. Configures itself to use the given proxy for all future network requests
+3. If snapd is installed, configures snapd to use the given proxy
+4. If Livepatch has already been enabled, configures Livepatch to use the given proxy
+   1. If Livepatch is enabled after this command, UA client will configure
+      Livepatch to use the given proxy at that time.
 
-The format for the proxy configuration values is:
+To remove HTTP/HTTPS proxy configuration, run the following:
 
-`<protocol>://[<username>:<password>@]<fqdn>:<port>`
+```console
+$ sudo ua config unset http\_proxy
+$ sudo ua config unset https\_proxy
+```
+
+After running the above commands, UA client will also remove proxy
+configuration from snapd (if installed) and Livepatch (if enabled).
+
+## APT Proxies
+
+APT proxy settings are configured separately. To have UA client manage your
+APT proxy configuration, run the following commands:
+
+```console
+$ sudo ua config set apt\_http\_proxy=http://host:port
+$ sudo ua config set apt\_https\_proxy=https://host:port
+```
+
+After running the above commands, UA client:
+
+1. Verifies that the proxy works by using it to reach `archive.ubuntu.com` or `esm.ubuntu.com`.
+2. Configures APT to use the given proxy by writing an apt configuration file to
+   `/etc/apt/apt.conf.d/90ubuntu-advantage-aptproxy`.
+
+> **Note**
+> Any configuration file that comes later in the apt.conf.d
+> directory could override the proxy configured by the UA client.
+
+To remove the APT proxy configuration, run the following:
+
+$ sudo ua config unset apt\_http\_proxy
+$ sudo ua config unset apt\_https\_proxy
+
+> **Note**
```

Comment:
Considering this is 27.10, could we add that `apt_http(s)_proxy` will be deprecated.

Reply:
Yes, I think we can improve on that, but let's use a separate PR for this. The main goal of this PR is just to align on what we will release next and not to provide any extra functionality

Reply:
Okay. No problem.